packages/happy-dom/src/event/events/AnimationEvent.ts

```diff
@@ -0,0 +1,23 @@
+import Event from '../Event';
+import IAnimationEventInit from './IAnimationEventInit';
+
+export default class AnimationEvent extends Event {
```

Comment:
Can we add AnimationEvent to the main index.ts file as well?

Reply:
Fixed! 👍🙂